src/include/server_limits.h

```diff
@@ -169,7 +169,11 @@ enum PBS_Daemon_Protect {
 	PBS_DAEMON_PROTECT_OFF,
 	PBS_DAEMON_PROTECT_ON
 };
+#ifdef WIN32
+void daemon_protect(int, enum PBS_Daemon_Protect);
+#else
 void daemon_protect(pid_t, enum PBS_Daemon_Protect);
+#endif
```

Comment:
I discussed this with Nithin during review. All projects that link to Python, try not to re-define pid_t. include projects doesn't link with python and hence the definition is not available to it. I thought a better approach would be to include win.h here. This would avoid defining pid_t at multiple places. This would require minor modification in win.h.